services/admin/src/store.test.ts

```diff
@@ -105,3 +105,23 @@ test('getAllTranscribedValues', () => {
   store.addAdjudication('chief-of-staff', cvrId, '');
   expect(store.getAllTranscribedValues()).toEqual(['Daffy', 'Mickey Mouse']);
 });
+
+test('getAdjudicationCountsByContestIdAndTranscribedValue', () => {
+  const store = Store.memoryStore();
+  const cvrId = store.addCvr('test');
+  store.addAdjudication('mayor', cvrId, 'Mickey Mouse');
+  store.addAdjudication('mayor', cvrId, 'Mickey Mouse');
+  store.addAdjudication('county-commissioner', cvrId, 'Daffy');
```

Comment:
We could add some records to also cover:

- Same contest, different transcribed values
- Same transcribed value, different contests

To make sure the grouping logic is working exactly as intended (and not grouping the above)